src/main/java/ru/teti/springcourse/dao/PersonDAO.java

```diff
@@ -1,51 +1,31 @@
 package ru.teti.springcourse.dao;
 
-import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
 import ru.teti.springcourse.models.Person;
 
-import java.sql.*;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.PreparedStatement;
+import java.sql.DriverManager;
 import java.util.ArrayList;
 import java.util.List;
 
-@Component
-public class PersonDAO {
-    private static int PEOPLE_COUNT;
-
-    private static final String URL = "jdbc:postgresql://localhost:5432/first_db";
-    private static final String USERNAME = "postgres";
-    private static final String PASSWORD = "Strong1+7";
-
-    private static Connection connection;
+import static ru.teti.springcourse.dao.DbConstants.*;
 
-    static {
-        try {
-            Class.forName("org.postgresql.Driver");
-        } catch (ClassNotFoundException e) {
-            e.printStackTrace();
-        }
-
-        try {
-            connection = DriverManager.getConnection(URL, USERNAME, PASSWORD);
```

Comment:
As we need to close all resources after their usage done, it would be better to move getting of the DB connection into a separate method and use it everywhere it needed within try-with-resources block. Or implement autocloseable interface. Its up to you.